docs/ckb-core-dev.md

```diff
@@ -60,9 +60,13 @@ cycle_length = 12
 ```
 [pow]
 func = "Dummy"
+```
+
+and don't forget to modify `ckb-miner.toml` miner workers section:
 
-# Delay offset (in milliseconds)
-[pow.params.delay]
-type = "constant"
-value = 5000
+```
+[[miner.workers]]
+worker_type = "Dummy"
```

Comment:
I think different types do not allow to exist at the same time, pow type should be global

Reply:
pow type is global, but the user may need to configure different worker type, for example, running 1 miner on CPU and two miners on different GPUs:

```
[[miner.workers]]
worker_type = "CuckooCpu"
threads     = "2"

[[miner.workers]]
worker_type = "CuckooGpuLean"
device_id   = "..."

[[miner.workers]]
worker_type = "CuckooGpuMean"
device_id   = "..."
```

Reply:
I don't think such `Dummy` here is served for your purpose. It's different level

Reply:
@zhangsoledad I'm going to stick on this commit: https://github.com/nervosnetwork/ckb/commit/4059a26365b3cda571ebc7d5cfdae9ca62662d0c after fighting with `toml::Value`, please review again, thanks.